internal/app/orchestrator/orchestrator.go

```diff
@@ -191,24 +179,24 @@ func (o *orchestrator) CreateWatch(req transport.Request) (transport.Watch, func
 		o.logger.With("error", err).With("aggregated_key", aggregatedKey).Warn(ctx, "failed to fetch aggregated key")
 	}
 
+	cachedResponse := false
```

Comment:
is this the same as the `cached.Resp != nil` check that we use in the next line? Can we just reuse that rather than introduce a new var?

Reply:
thats right..changed it to use a single variable.